### DISCUSSION TITLE
not-prose only on child elements

### DISCUSSION THREAD

**Discussion by JurjenRoels** [tailwindlabs/tailwindcss#11773] [Help] [answered]
I hav a question about Tailwindcss and specific the not-prose class. 

I want an image to not be formatted by prose. so i have `<img class=“not-prose”>`

 But for the image not to be shown in prose markup I have to have a parent tag with class=“not-prose”. So `<div class=“not-prose”><img /></div>` or `<td class=“not-prose”><img></td>` is working. <img class=“not-prose”> is not.  

This is the class in the inspector that is matched.`(.prose :where(img):not(:where([class~=not-prose] )) {…..}) `

Do I need to add some classes to my safe list in compiling the css file or is this the intended behavior?

Thank you.

**Comment by wongjn** [comment, marked as answer]
It is intended behavior that you need to wrap the non-prose-styled content with an element that has the `not-prose` class, [as per the documentation]():

> If you have a block of markup embedded in some content that shouldn’t inherit the `prose` styles, use the `not-prose` class to sandbox it:
> ```html
> <article class="prose">
>   <h1>My Heading</h1>
>   <p>...</p>
> 
>   <div class="not-prose">
>     
>   </div>
> 
>   <p>...</p>
>   
> </article>
> ```

**Reply by JurjenRoels** [reply]
Thank you.

**Reply by piszczu4** [reply]
What if I want to again add prose to child that is wrapped in `not-prose`? It seems that this child prose is ignored beacuse of parent `not-prose` :(
